### ISSUE TITLE
Last version build error

### ISSUE THREAD

**Issue by blueiceprj** [stevermeister/ngx-cookie-service#203]
### Describe the bug

./node_modules/ngx-cookie-service/fesm2020/ngx-cookie-service.mjs:1:0-61 - Error: Module not found: Error: Can't resolve '@nguniversal/express-engine/tokens' in '......../node_modules/ngx-cookie-service/fesm2020'

### Steps to Reproduce

Build

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

no

### Expected behaviour

Build

### What version of the library you see this issue?

last

### What version of Angular are you using?

Angular 13

**Comment by pavankjadda** [comment]
Do you see this error when using the latest version and building the app?

**Comment by Vorlif** [comment]
We have a bug similar to this one, also in version 13.1.0. We do not use SSR.

```
Error: node_modules/ngx-cookie-service/lib/cookie.service.d.ts:1:25 - error TS7016: Could not find a declaration file for module 'express'. '/path/node_modules/express/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/express` if it exists or add a new declaration (.d.ts) file containing `declare module 'express';`

1 import { Request } from 'express';
```

**Comment by pavankjadda** [comment]
@Vorlif @blueiceprj try version 13.1.1 and let me know of it fixes the issue

**Comment by Vorlif** [comment]
Unfortunately, no change.

**Comment by pavankjadda** [comment]
> @pavankjadda Unfortunately, no change.

Please delete the node modules and install again. Version `13.1.1` should fix the error

**Comment by pavankjadda** [comment]
> @pavankjadda Unfortunately, no change.

Okay. I understand the issue. Published new version `13.1.2` to revert the changes. I tested in new project, it works fine.
